src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -1393,13 +1393,14 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
 
                 foreach (var entry in accessRuleChange.Entries)
                 {
-                    var entryLookup = new AccessRuleEntryLookup(lookup.EntityId, entry.Key.RoleKey);
+                    var entryLookup = new AccessRuleEntryLookup(lookup.EntityId, entry.Key.RoleKey, entry.Key.ObjectModuleId.ToString());
```

Comment:
`ObjectModuleId` is an enum and we should map it as such in our database.

Reply:
addressed